admin/aioseop_module_class.php

```diff
@@ -1504,266 +1504,442 @@ function get_the_image_by_post_thumbnail( $p = null ) {
 				return $image[0];
 		}
 
+		/**
+		 * Gets the image by attachment.
+		 * @since 1.0.0
+		 * @param String $p.
+		 * @return array.
+		 */
 		function get_the_image_by_attachment( $p = null ) {
-
-			if ( $p === null ) {
+			if ( $p === null )	{
 				global $post;
 			} else {
 				$post = $p;
 			}
-
-			$attachments = get_children( Array( 'post_parent' 		=> $post->ID,
-												'post_status' 		=> 'inherit',
-												'post_type' 		=> 'attachment',
-												'post_mime_type'	=> 'image',
-												'order' 			=> 'ASC',
-												'orderby' 			=> 'menu_order ID' ) );
-
+			$attachments = get_children( Array(
+					'post_parent' 		=> $post->ID,
+					'post_status' 		=> 'inherit',
+					'post_type' 		=> 'attachment',
+					'post_mime_type'	=> 'image',
+					'order' 			=> 'ASC',
+					'orderby' 			=> 'menu_order ID'
+				)
+			);
 			if ( empty( $attachments ) ) {
 				if ( 'attachment' == get_post_type( $post->ID ) ) {
 					$image = wp_get_attachment_image_src( $post->ID, 'large' );
 				}
 			}
-
-			/* If no attachments or image is found, return false. */
 			if ( empty( $attachments ) && empty( $image ) )
 				return false;
-
-			/* Set the default iterator to 0. */
 			$i = 0;
-
-			/* Loop through each attachment. Once the $order_of_image (default is '1') is reached, break the loop. */
-			foreach ( $attachments as $id => $attachment ) {
-				if ( ++$i == 1 ) {
+			foreach ( $attachments as $id => $attachment )	{
+				if ( ++$i == 1 )	{
 					$image = wp_get_attachment_image_src( $id, 'large' );
-					$alt = trim( strip_tags( get_post_field( 'post_excerpt', $id ) ) );
+					$alt = trim(
+						strip_tags(
+							get_post_field(
+								'post_excerpt',
+								$id
+							)
+						)
+					);
 					break;
 				}
 			}
-
-			/* Return the image URL. */
```

Comment:
@joelrd @amostajo Same thing here. If we're going to remove the comments, it should at least be represented in the docblock.